packages/expo-app-auth/src/__tests__/AppAuth-test.ts

```diff
@@ -0,0 +1,55 @@
+import { mockProperty, unmockProperty } from 'jest-expo';
+
+import ExpoAppAuth from '../ExpoAppAuth';
+import * as AppAuth from '../AppAuth';
+
+describe('AppAuth', () => {
```

Comment:
Don't need a top-level `describe`, can reduce all this by one level of indentation